app/configurations/realtimeUtils.js

```diff
@@ -254,6 +254,19 @@ export default {
 
     routeSelector: defaultRouteSelector,
 
+    active: true,
+  },
+  hb: {
+    mqttTopicResolver: function mqttTopicResolver() {
+      return '/gtfsrt/vp/#';
+    },
+
+    mqtt: 'wss://api.staging.stadtnavi.eu:1884',
```

Comment:
The URL has changed and should be `wss://api.stadtnavi.de/mqtt`

Reply:
Done

Reply:
Is it? I still see `wss://api.staging.stadtnavi.eu:1884`.

Can you re-check?

Reply:
My bad, I reverted the change accidentally. Although the `wss://api.stadtnavi.de/mqtt` does not return any data yet.

Reply:
Hmm, maybe you need the trailing slash, so `wss://api.stadtnavi.de/mqtt`